index.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+const circuitbreaker = require('circuitbreaker');
+
+class CircuitBreaker {
+
+    /**
+     * Construct a CircuitBreaker object
+     * @method constructor
+     * @param  {Function}   command              The command to be running that requires the circuit breaker
+     * @param  {Object}     [options]            Options to configure the circuit breaker with
+     * @param  {Number}     options.timeout      The timeout in ms to wait for a command to complete
+     * @param  {Number}     options.maxFailures  The number of failures before the circuit is tripped
+     * @param  {Number}     options.resetTimeout The timeout in ms to reset the switch
+     */
+    constructor(command, options) {
+        this.command = command;
+        this.options = options || {
```

Comment:
So this means if `options = { timeout: 200 }`, it will set `this.options = { timeout: 200 }` without setting maxFailures and resetTimeout to the default values (5 and 50). Do we want this behavior? If so, we should make it clear in the documentation.